gravitee-am-reporter/gravitee-am-reporter-api/src/main/java/io/gravitee/am/reporter/api/audit/model/AuditEntity.java

```diff
@@ -57,6 +57,10 @@ public class AuditEntity {
      */
     private String referenceId;
 
+    private String externalId;
```

Comment:
Those two fields are very specific to a User and not the others type of Audits (App, Domain, IdP, ...) I'm not sure this is the best solution to add the values at the root level of an Audit ...

Reply:
I conscious that by using the attributes it may be difficult to search / filters on those values ...